### PR TITLE
Adding radical.gtod

### PR DESCRIPTION
**Pull request by mtitov**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mtitov** [comment]
> GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

@andre-merzky Andre, please confirm here

**Review by andre-merzky — APPROVED** [review]
LGTM

**Comment by mtitov** [comment]
@conda-forge-admin, please ping team

File: recipes/radical.gtod/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "radical.gtod" %}
+{% set version = "1.13.0" %}
+{% set sha256 = "15df4ae728a8878b111cfdedffb9457aecc8003c2cfbdf2c918dfcb6b836cc93" %}
```
Comment (ocefpaf) [inline recipes/radical.gtod/meta.yaml:3]:
```suggestion
```

File: recipes/radical.gtod/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (ocefpaf) [inline recipes/radical.gtod/meta.yaml:11]:
```suggestion
  sha256: 15df4ae728a8878b111cfdedffb9457aecc8003c2cfbdf2c918dfcb6b836cc93
```

File: recipes/radical.gtod/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```
Comment (ocefpaf) [inline recipes/radical.gtod/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

**Review by ocefpaf — COMMENTED** [review]
The excess of jinja makes the recipe a bit hard to read and does not bring any advantage. Please try grayskull to create the recipes b/c it will bring it in a format that requires minimum edits before we merge.

**Comment by mtitov** [comment]
@ocefpaf thank you! it is ready

**Pull request merged**